doc/source/notebooks/advanced/het_gpr_script.ipynb

```diff
@@ -0,0 +1,512 @@
+      "text/plain": [
+       "<Figure size 1080x360 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "mean, var = base_model.predict_y(X2)\n",
+    "plot_distribution(X2, Y2, mean.numpy(), np.sqrt(var.numpy()))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6179120e",
+   "metadata": {},
+   "source": [
+    "When we have more data, we can afford to fit a more complex likelihood model. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "b9be33d9",
+   "metadata": {
+    "lines_to_next_cell": 2
+   },
+   "outputs": [],
+   "source": [
+    "class QuadraticLikelihood(MultiLatentTFPConditional):\n",
+    "    \"\"\" Does what it says on the tin.  \"\"\"\n",
+    "    \n",
+    "    def __init__(self, ndims: int = 1,  **kwargs):\n",
+    "\n",
+    "        self.noise_gradient = Parameter(np.ones(ndims), prior=GRADIENT_PRIOR)\n",
+    "        self.noise_curvature = Parameter(np.ones(ndims), prior=GRADIENT_PRIOR)\n",
+    "        self.constant_noise = Parameter(1.0, transform=positive(lower=MIN_NOISE_SCALE))\n",
```

Comment:
As above, I'm not sure how much this positive transform really is the right way to solve the problem...